client/components/App.js

```diff
@@ -65,12 +77,44 @@ const App = () => {
       >
         <MyProvider>
           <PageLayout>
-            <ProductsPage />
+            {selection.length > 0 ? (
+              <ProductsPage productIds={selection} />
+            ) : (
+              <Page>
```

Comment:
I wonder if we want to make this a component as well, like `EmptyStatePage` or some such?

Reply:
ooh i like that!